### ISSUE TITLE
Stage displaying a PDF rendition in the html viewer

### ISSUE THREAD

**Issue by gsteimer** [tsgrp/HPI#1344]
If a document has both HTML and PDF renditions, it looks like HPI is trying to display the PDF inside the HTML viewer:

![image](https://cloud.githubusercontent.com/assets/1377228/24312372/39c613b6-10a6-11e7-994d-835425d0bc23.png)

The document above has both an HTML and a PDF rendition:

![image](https://cloud.githubusercontent.com/assets/1377228/24312356/2b9092a8-10a6-11e7-9deb-2c336a0c0751.png)

To recreate the issue above, simply use the `Send Email` action on the insurance trac.

From talking to @benallenallen - we think this may be occuring due to the changes we made to prefer the image viewer recently.   If a document has a PDF rendition for HTML primary content, we should be displaying the PDF in the proper PDF viewer (either OA or PDF.js).  The HTML viewer should only be used for documents that are native HTML with no PDF rendition.

**Comment by gsteimer** [comment]
Note - to test the above, you may need to test using Alfresco since we use `tsg:renditioned`.  We probably do not have autorender turned on for HTML in our internal docbases.

**Comment by grahamsinger** [comment]
![image](https://cloud.githubusercontent.com/assets/8784597/24472375/1bf3232c-148b-11e7-8038-d792e1a67554.png)

Should the order of these viewers be changed so that `OpenAnnotate` is "before" the `HTML Viewer` and then changing the current priorities so that OpenAnnotate has a higher one than the HTML viewer? @gsteimer @benallenallen

**Comment by gsteimer** [comment]
I can't think of a scenario where we'd prefer the HTML viewer to OA.  The HTML viewer should only be used for content that has HTML content with no PDF rendition.  @benallenallen - do you agree?

**Comment by benallenallen** [comment]
Yes, that seems sensible to me... no idea why it isn't just right next to PDF.js (After PDF.js)... that grid is hard to follow, but the theory was that left to right is the same priority that is configured in the individual modules, so i would expect:

![image](https://cloud.githubusercontent.com/assets/4850976/24473039/8b43f40c-148d-11e7-914c-5ac2d7b73ec3.png)

Can we add some borders or something to make that clearer???

1. PDF.js
2. OpenAnnotate
3. Image Viewer
4. HTML Viewer
5. Native
6. Download

**Comment by gsteimer** [comment]
@benallenallen - didn't we change it so the image viewer was preferred first?

**Comment by benallenallen** [comment]
See my updated comment above... changed the order:

![image](https://cloud.githubusercontent.com/assets/4850976/24473479/0a3ea238-148f-11e7-9fe2-012b1059f111.png)

**Comment by grahamsinger** [comment]
Committed revision 17655.

**Comment by grahamsinger** [comment]
Made sure that we arent trying to stream a PDF rendition to the HTML viewer.  Also changed the default priorities of the viewers so that OA is now a higher priority than the HTML viewer.  Unit tested as well.
